examples/ignition/install-reboot.yaml

```diff
@@ -1,20 +1,31 @@
 ---
 systemd:
   units:
-    - name: install.service
+    - name: installer.service
       enable: true
       contents: |
         [Unit]
         Requires=network-online.target
         After=network-online.target
         [Service]
-        Type=oneshot
-        ExecStart=/usr/bin/curl {{.ignition_endpoint}}?{{.query}}&os=installed -o ignition.json
-        ExecStart=/usr/bin/coreos-install -d /dev/sda -C {{.coreos_channel}} -V {{.coreos_version}} -i ignition.json {{if index . "baseurl"}}-b {{.baseurl}}{{end}}
-        ExecStart=/usr/bin/udevadm settle
-        ExecStart=/usr/bin/systemctl reboot
+        Type=simple
+        ExecStart=/installer
+        Restart=always
```

Comment:
Please don't restart. Chances are good that it's just going to fail again and again. Plus, it will mask any failures that occur.